hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -190,6 +192,30 @@ public BasicRootedOzoneClientAdapterImpl(String omHost, int omPort,
       this.configuredDnPort = conf.getInt(
           OzoneConfigKeys.DFS_CONTAINER_IPC_PORT,
           OzoneConfigKeys.DFS_CONTAINER_IPC_PORT_DEFAULT);
+
+      // Fetches the bucket layout to be used by OFS.
```

Comment:
Please extract this entire bucket layout init logic to a method, like,

`initDefaultBucketLayout(conf);`

Reply:
Done